<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFuncDlg</class>
 <widget class="QDialog" name="QFuncDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Function</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Function</string>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>(x-2)^4+(x-2y)^2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>(x1-2x2)^2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>(x2-x1^2)^2+100*(1-x1)^2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>x^2+y^2-y+x^3</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>241</width>
     <height>112</height>
    </rect>
   </property>
   <property name="title">
    <string>Area</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="l_x">
       <property name="text">
        <string>&amp;x</string>
       </property>
       <property name="buddy">
        <cstring>x1Edit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="x1Edit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l12">
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="x2Edit">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="l_y">
       <property name="text">
        <string>&amp;y</string>
       </property>
       <property name="buddy">
        <cstring>y1Edit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="y1Edit">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l34">
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="y2Edit">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget_2</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>241</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>128</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="canselButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>canselButton</sender>
   <signal>clicked()</signal>
   <receiver>QFuncDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
